easybuild/easyconfigs/b/Blitz++/Blitz++-0.10-foss-2016a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'Blitz++'
+version = '0.10'
+
+homepage = 'http://blitz.sourceforge.net/'
+description = """Blitz++ is a (LGPLv3+) licensed meta-template library for array manipulation in C++
+ with a speed comparable to Fortran implementations, while preserving an object-oriented interface"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = ['blitz-%(version)s.tar.gz']
+source_urls = ['https://sourceforge.net/projects/blitz/files/blitz/%(name)s %(version)s',
+               'download']
```

Comment:
This is/looks wrong... is there really supposed to be a space in between name and version? Should it be `%(name)s/%(version)s`?

Also, you need to wrap this in a tuple `(...)`, otherwise you have a list of two candidate source URLs, rather than one where `/download` is being glued at the end:

```
source_urls = [('https://sourceforge.net/projects/blitz/files/blitz/%(name)s/%(version)s', 'download')]
```

You can update this PR after fixing remarks by just using `eb --update-pr 2784 Blitz++-0.10-foss-2016a.eb`, after editing `Blitz++-0.10-foss-2016a.eb` to fix the remarks.